### DISCUSSION TITLE
Audio File is missing- Copy URL not Working

### DISCUSSION THREAD

**Discussion by ariya888** [aclap-dev/video-downloadhelper#93] [Help and Q&A]
### Your question

In previous Version, we could copy URL of Audio file only but in this new Version, this option is gone, plus, Copy URL is not Working Please Fix it. Thank you

**Comment by paulrouget** [comment]
Please share your version and browser. Can't help you otherwise: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F
